src/client/web/WebRTC.ts

```diff
@@ -0,0 +1,114 @@
+import Peer from 'peerjs'
+import Network from '../services/Network'
+
+export default class WebRTC {
+  private myPeer: Peer
+  private peers = {}
+  private videoGrid = document.getElementById('video-grid')
+  private buttonGrid = document.getElementById('button-grid')
+  private myVideo = document.createElement('video')
+  private myStream?: MediaStream
+
+  constructor(userId: string, network: Network) {
+    this.myPeer = new Peer(userId)
+
+    // mute your own video stream (you don't want to hear to yourself)
```

Comment:
nit:     // mute your own video stream (you don't want to hear ~to~ yourself)

Reply:
LOL. Bad bad Chinglish.